metaci/release/tests/test_utils.py

```diff
@@ -13,22 +13,25 @@ def test_send_release_webhook(mocked_responses, mocker, transactional_db):
         METACI_CHANGE_CASE_URL_TEMPLATE="{case_id}",
         METACI_RELEASE_WEBHOOK_ISSUER="MetaCI",
         METACI_RELEASE_WEBHOOK_AUTH_KEY="test",
+        GUS_BUS_OWNER_ID="00G",
     )
     mocked_responses.add(
-        "POST", "https://webhook/release/", json={"success": True, "id": "2"}
+        "POST",
+        "https://webhook/release/",
+        json={"success": True, "implementationSteps": ["a2dAG0000000"], "id": "2"},
     )
 
     project_config = Mock(project__package__name="Test Package")
     project_config.get_version_for_tag.return_value = "1.0"
     release = ReleaseFactory()
 
-    send_release_webhook(project_config, release, "INFRA.instance1")
+    send_release_webhook(release, "INFRA.instance1")
 
     assert release.change_case_link == "2"
```

Comment:
Do we need an assertion to make sure that the ids from implementationSteps were stored on the correct ImplementationStep models? They need to be there so that the start/stop automation can find the right id.